src/server/lib/fetch.ts

```diff
@@ -6,7 +6,7 @@
  * Solution taken from: https://github.com/node-fetch/node-fetch/issues/1279#issuecomment-915063354
  */
 
-import { RequestInfo, RequestInit, Response } from 'node-fetch';
+import type { RequestInfo, RequestInit, Response } from 'node-fetch';
```

Comment:
not seen `import type` before, just wondering why we're changing it here

Reply:
This is related to `node-fetch` being a pure ESM package and us importing it asynchronously. There is a comment here on the same thread linked in the comments in this file that explains - https://github.com/node-fetch/node-fetch/issues/1279#issuecomment-915061092

Reply:
ahh thanks, that's really interesting